test/common.py

```diff
@@ -0,0 +1,73 @@
+""" Functionailty that is common for several tests, and therfore stored in a
+common module.
+"""
+import numpy as np
+
+import porepy as pp
+
+
+def setup_flow_assembler(gb, method, data_key=None, coupler=None):
+    """ Setup a standard assembler for the flow problem for a given grid bucket.
```

Comment:
Good!

Reply:
This should maybe go into the new folder we discussed on Thursday. It would make @alessiofumagalli  happy

Reply:
I agree. But I'm not entirely convinced about "mortar_solution". Since the node variable is "pressure", it would be more consistent using something more physics specific. "flux" and "mortar_flux" come to mind.